### ISSUE TITLE
`No such file or directory` with non-ASCII character in path

### ISSUE THREAD

**Issue by mischnic** [DoctorEvidence/lmdb-js#144]
1. Create & cd into a folder named `Benoît`
2. Add this script inside of that folder
```js
const { open } = require("lmdb"); // or require
let myDB = open(__dirname + "/.parcel-cache", {
	name: "parcel-cache",
	compression: true,
});
```
3. 
```
Benoît/node_modules/lmdb/dist/index.cjs:1943
		lmdbError(rc);
		^

Error: No such file or directory
    at open (Benoît/node_modules/lmdb/dist/index.cjs:1943:3)
    at Object.<anonymous> (Benoît/x.js:2:12)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  code: 2
}
```

**Comment by kriszyp** [comment]
Fix published in v2.2.4 (hopefully).

**Comment by mischnic** [comment]
Thanks!
